.travis.yml

```diff
@@ -0,0 +1,23 @@
+services:
+- docker
+env:
+  global:
+    secure: m/S6viN8qIJM6qCntGch+HmP8gTA/YdZtyXfz3EvtLEC8FL5D7e64Ulde/cS8i4TAg380LmjY+vuHOk29DjMG3otFTG0J742/tmQI0b+uHFQSuFwj6kWl12yX78Kqa6gZhAnukMPEaFmFX7DwCFFYL+oj7Kx/NQIQtPd0nTDzWlFd2lxtPcgOgvB4fXaVAVZD4Y21K30joQ7AdfVIhBmKPq49YaLeC1g9yMtzeZMvOIoodvr5RWL1rUhGv8XPCORRhLz/SThqG+c4n0BZgnDEqeMI6+RTSuu9tIZUsgoN2LWBLDwEgGFQ4KkhtMlW37CQZ8BFaVynmguqoHdhmAM6jA3HJb3DmJUqu5ogD0lmobiNhi26SYrU/aKDUTtNsbIL2fOKEZx7JmChoDIUjMiLsLQj6AVVZseBAQ0YXtp7MzynQBtjfeSjyYu3C3H26PaGn9L9PmLPGxR176Z4MihF1iXQ7AfXwxoyApwhlqFUPEaCtlQFe6M+gAznO3G5BfEsovsEIutxHS6AhVilnnwHwsHtseXZXqDCskxAkqL83Rhrgv3eaeraetsC5AQT/cXoWTFtCMdnwgMXh0iljKbpTzAGZjiIqBY7uiReWvOMai6a9L+75rklOQLPB0atur9uQ6XeoI2dUi6YQ4fEdH6GJvgjy9LmRoK6GhWVvOrZwg=
+    secure: S1m5nQUOS0pu4R66kNUTTcVMGqShMYMQwYrf7aXZ4GF1XDsuMdvT+OnJ716YTiVxHi4FqSWl+OQUn59aq1HGqHSAJjJfZMaPhrbUkp4ufGp9FyXGDy3xB0bCd/OACiDOuMom/ZaaPx6urEEqSdqeJFDsbM1v9w1mWrRWmCXpbgauWDW+AJcTfD/o2JCBztz9AOeZDY2MbcKtCRpBt+QUJx7REGN3tdjSgz6iK+OxEah3+jdhrmRDW8pBcplqPm35mUBSFnqNohqfBc3EBd5Y20DcriWaaUsV9dz6oy5QTFM5yzZlW+PZ/GCljY75N3GR9mI/C4LlQaE6VswfFZyqfOhtzAX+vDE/Aa1JF+he1pemQHBVdyyWPQIh5yhWhmGt6ojtOkQBFo2CCUpLc5oLIYw0wP2hNsikSRqlK8qvMCO6PaIokwKVHBFlFtmMsMBMvek7w6gQ84JqbBVlNcH54uT5InMIycgYM2SKlWn533z/9clx85vnIpGJyumCHYl53ncqcG4cC3NozME2B+xEU7wJLuBzFk4T4Y1hnG9T2J0xANaovy5WL0N2E19ji4U3kCQPbMN19rUCVsJXdTs7IHs2RJZyeLrImbAILL1r5zYq6vaq2HolG827IQKqQECJ2+rYABxCWq6+6WScexEbpkXtRGhecza2rtQlpuUrj4M=
+
+script:
+    - docker build --target test --tag my-test-image .
+    - docker docker run my-test-image tests/test_view_model.py
+    - docker docker run my-test-image tests/test_integration.py
+    - docker docker run my-test-image tests/test_selenium.py
```

Comment:
You're not actually running your tests... you have an extra `docker` at the start of the commands.

The build passes, because the docker command line tool isn't returning an error code, but if you look at the build logs, the tests aren't running and in response, the docker CLI is telling you `Usage:	docker [OPTIONS] COMMAND`

Reply:
In order to get your selenium test to pass, you will need to pass the environment variables from the Travis environment into the docker container. So you will need to run something like `docker run -e TRELLO_API_KEY -e TRELLO_API_TOKEN my-test-image tests/test_selenium`

(You used `--env-file ./.env` when running it locally but can't do that in Travis because it doesn't have a .env file)